utils/generateThemeClass.js

```diff
@@ -0,0 +1 @@
+export default prop => `mdc-theme--${prop}-bg `;
```

Comment:
can we call it ThemeUtils? so that we can put more stuff in here in future?

Reply:
Capital `T` or lowercase `t`? I thought the precedent was for capitals to be reserved for components.

Reply:
ohyeah u r right. themeUtils.js